### PR TITLE
A64fx intrinsics [a64fx]

### PR DESCRIPTION
**Pull request by camierjs**
Adds the intrinsics for the Fujitsu's [A64FX](https://github.com/fujitsu/A64FX) architecture.

Tested on Fugaku with the HPC templated code.

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#2113](https://github.com/mfem/mfem/pull/2113) | @camierjs | @tzanio | @tzanio + @v-dobrev | 03/21/21 | 03/22/21 | 03/25/21 | |

### CODE REVIEW

**Comment by camierjs** [comment]
LGTM

**Comment by tzanio** [comment]
This PR is now [under review](https://github.com/mfem/mfem/blob/master/CONTRIBUTING.md#pull-requests) (see the table in the PR description). To help with the review process, please **do not force push** to the branch.

**Review by tzanio — APPROVED** [review]
Thank you @camierjs! 🚀

File: linalg/simd.hpp [hunk 1]
```diff
@@ -17,7 +17,10 @@
 // --- AutoSIMD + specializations with intrinsics
 #include "simd/auto.hpp"
 #ifdef MFEM_USE_SIMD
-#if defined(__VSX__)
+#if defined(__aarch64__) && defined(__ARM_FEATURE_SVE)
+#warning simd/sve.hpp
```
Comment (v-dobrev) [inline linalg/simd.hpp:21]: Do we really want this warning?

File: linalg/simd/sve.hpp [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_SIMD_SVE_HPP
+#define MFEM_SIMD_SVE_HPP
+
+#if defined(__aarch64__) && defined(__ARM_FEATURE_SVE)
+
+#include "../../config/tconfig.hpp"
+#include <arm_sve.h>
+#define PG svptrue_b64()
```
Comment (v-dobrev) [inline linalg/simd/sve.hpp:19]: `PG` is a very generic name for a macro. At least add an `MFEM_` prefix; consider expanding `PG` as well. What does it stand for?

Comment (camierjs) [reply linalg/simd/sve.hpp:19]: It's for the predicate group - or mask. I'll expand it, that will be easier.

File: linalg/simd.hpp (hunk 1 above)
Comment (camierjs) [reply linalg/simd.hpp]: `removed`

File: linalg/simd/sve.hpp (hunk 2 above)
Comment (camierjs) [reply linalg/simd/sve.hpp]: `done`

**Comment by camierjs** [comment]
LGTM

**Review by v-dobrev — APPROVED** [review]

**Comment by tzanio** [comment]
Merged in `next` for testing...

**Pull request merged**